uwblueprint/childrens-aid-society
# Implement ‘Add Child’ Information page

## Notion ticket link

[Implement ‘Add Child’ Information page](https://www.notion.so/uwblueprintexecs/Implement-Add-Child-Information-page-57672d8f7dee49ee8798d27fcd45fe33)



## Implementation description
* Use Formik to create a child information form



## Steps to test
1. [Link to Figma](https://www.figma.com/file/1obAgfYZ639hpKXV1ZP7hJ/3.-FeatureSets-%3E-Case-intake?node-id=1175%3A19815&t=worHmYGXW7lNMGGU-0)
2. Click on link to deployed web app 
3. Navigate to `\intake`
4. Click next button until Individual Details page
5. Click on add child button
6. Select Child information
7. Verify field names and placeholders match Figma file 


## What should reviewers focus on?


## Checklist
